src/towncrier/newsfragments/374.feature.rst

```diff
@@ -0,0 +1,2 @@
+
+Add -m flag to create endpoint
```

Comment:
This is the end user text that helps advertise the new functionality.

I think that is best to write it as a full sentence.

```suggestion
The `towncrier create` command line now has a new `-m TEXT` argument that is used to define the content of the newly created fragment.
```